File: Java/Exercicio3/MembrosEstaticosEAbstratos/src/Assistente.java
public class Assistente extends Funcionario {
    public Assistente(String nome, String CPF, double salarioBase) {
        super.setNome(nome);
        super.setCPF(CPF);
        super.setSalarioBase(salarioBase);
    }

    @Override
    public double calculaSalario() {
        return super.getSalarioBase();
    }
}

File: Python/Funcionario/Assistente.py
from Funcionario import Funcionario

class Assistente(Funcionario):
    def __init__(self, nome, cpf, salario_base):
        super().__init__(nome, cpf, salario_base)        
        self.salario = salario_base
    
    def calculaSalario(self):
        return self.salario